RetailCoder.VBE/UI/FindSymbol/FindSymbolDialog.cs

```diff
@@ -26,6 +26,8 @@ private void viewModel_Navigate(object sender, NavigateCodeEventArgs e)
         public FindSymbolDialog()
         {
             InitializeComponent();
+
+            this.Text = string.Format("Rubberduck - {0}", RubberduckUI.FindSymbolDialog_Caption);
```

Comment:
Why should you want to have Rubberduck in the translated string if does not change?
If you feel that it is better to have it in the translated string. Let me know then I will change it.